rocketpy/Rocket.py

```diff
@@ -165,8 +165,12 @@ def __init__(
         radius,
         distanceRocketNozzle,
         distanceRocketPropellant,
-        powerOffDrag,
-        powerOnDrag,
+        powerOffDrag=None,
+        powerOnDrag=None,
+        rocket_length=None,
+        speed_of_sound=340.293988026089,
```

Comment:
Is it really necessary using so many decimal places? I mean, what a precise number you got here... hahahahahaha